kuma/wiki/helpers.py

```diff
@@ -318,22 +320,20 @@ def devmo_url(context, path):
     Create a URL pointing to Kuma.
     Look for a wiki page in the current locale, or default to given path
     """
+    fallback_locale = settings.WIKI_DEFAULT_LANGUAGE
+
     if hasattr(context['request'], 'locale'):
         locale = context['request'].locale
     else:
-        locale = settings.WIKI_DEFAULT_LANGUAGE
```

Comment:
Could this be rewritten as `locale = getattr(context['request'], 'locale', fallback_locale)`?